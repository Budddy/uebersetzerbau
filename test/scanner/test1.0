/* ***** this is a comment 
this */ not 1eDH +*-=,:)(;.<>> 315 End of_file2
