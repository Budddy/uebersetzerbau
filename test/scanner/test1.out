not
num 493
+
*
-
=
,
:
)
(
;
.
<>
>
num 315
id End
id of_file2
